File: DemcliT_WCF/DemcliT_WCF/Models/Pago.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace DemcliT_WCF.Models
{
    public class Pago
    {
        public long IdPago { get; set; }
        public string CodRespuesta { get; set; }
        public DateTime FechaRegistro { get; set; }
        public string UserLogin { get; set; }
        public string IdUserRegistro { get; set; }
        public int IdCanal { get; set; }
        public string Canal { get; set; }
        public int CodVendedor { get; set; }
        public string CodOficina { get; set; }
        public string Oficina { get; set; }
        public int CodCliente { get; set; }
        public string Monto { get; set; }
        public int IdFranquicia { get; set; }
        public string Franquicia { get; set; }
        public string NumAutorizacion { get; set; }
        public string NumTarjeta { get; set; }
        public string EMail { get; set; }
    }
}

File: Demclit_App/src/app/models/pago.ts
export class Pago {
  
  fechaRegistro:  Date;
  idUserRegistro: string; //NH User
  idCanal: number; //Id grupo canal de venta
  canal: string; //nombre del canal
  codVendedor: number;
  codOficina: number;
  codCliente: number;
  monto: number;// valor a pagar => valisar decimales
  idFranquicia: number;
  franquicia: string;
  numAutorizacion: string; //numero de baucher 
  numTarjeta: string;
  eMail: string;
  idPago:  number;
  
}
